bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/ui/DashboardEntryLabelProvider.java

```diff
@@ -67,7 +67,7 @@ public Image getColumnImage(Object element, int columnIndex) {
             Project project = devModeOps.getDashboardProject(projectName);
 
             if (project != null) {
-                if (project.isGradle()) {
+                if (project.getBuildType() == Project.BuildType.GRADLE) {
```

Comment:
Isn't this still the wrong precedence?

Reply:
NM.. this is all calculated when building the build type enum so you don't have to think about this here right?